app/src/main/AndroidManifest.xml

```diff
@@ -11,14 +11,16 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".views.SplashScreen"
-                  android:theme="@style/SplashTheme">
+        <activity
+            android:name=".views.SplashScreen"
+            android:theme="@style/SplashTheme">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".views.NoticeList" />
```

Comment:
I actually don't know about our layout and which activity comes first or before but if from NoticeList you can go to Login  you should specify parentActivityName in Login activity or opposite cause tha navigation in app may crash.